habitat-lab/habitat/tasks/rearrange/__init__.py

```diff
@@ -49,5 +49,5 @@ def _try_register_rearrange_task():
         HabitatSimActions.extend_action_space("empty")
     if not HabitatSimActions.has_action("rearrange_stop"):
         HabitatSimActions.extend_action_space("rearrange_stop")
-    if not HabitatSimActions.has_action("changejoint_action"):
-        HabitatSimActions.extend_action_space("changejoint_action")
+    if not HabitatSimActions.has_action("humanoidjoint_action"):
```

Comment:
I think this will change after @ASzot Work on refactor of the action space. Make sure you sync with him regarding what gets merged first.

Reply:
I would merge this first since it is a smaller change. I can help extending the actions to this one afterwards